lit/math/exp.thorin

```diff
@@ -0,0 +1,24 @@
+// RUN: rm -f %t.ll ; \
+// RUN: %thorin %s --output-ll %t.ll
+// RUN: clang++ %t.ll -o %t -Wno-override-module
+// RUN: %t foo bar; test $? -eq 64
+
+.import math;
+.import mem;
+.import core;
+
+.let _32 = 4294967296;
+.let I32 = .Idx _32;
+.let f32 = (23,  8);
+.let f64 = (52, 11);
+.let F32 = %math.F f32;
+.let F64 = %math.F f64;
```

Comment:
This code (at least the f F definitions) is so common that we might want it directly in the dialect thorin files.

Reply:
I'd like to have this later on in the `%math` namespace. Once we allow this for everything and not just axioms:
```
.let %math.f32 = (23, 8);
...
```